src/infrastructure/content/IContentCommunicator.java

```diff
@@ -0,0 +1,26 @@
+package infrastructure.content;
+
+public interface IContentCommunicator {
+	/*
+	 * UI will call this method with one json formatted string argument
+	 * @param : userDetails - It is json string which will contain server ipAddress, username, image
+	 */
+	void initialiseUser(String userDetails);
+	
+	/*
+	 * UI will call this to paas the message of client to everyone else
+	 * @param : message - it is the actual message as a string
+	 */
+	void sendMessageToContent(String message);

```

Comment:
I discussed with UI that all data is passed in JSON format. It is much easier to accomodate additional parameters when JSON is used. Suppose emoticon needs to be send, with JSON just a new field needs to be added